scripts/set_gradle_proxy.sh

```diff
@@ -38,6 +38,7 @@ writeProxyStringToGradleProps () {
     fi
 
     local PORT=${PROXY##*:}
+    PORT=${PORT/%\//}
```

Comment:
Commit message nit: It still talks about plural "slashes" there.

Reply:
Oups